src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -190,6 +190,8 @@ enum {
     /* Device supports indirect xgvmi MR. This flag is removed if xgvmi access
      * command fails */
     UCT_IB_MLX5_MD_FLAG_INDIRECT_XGVMI       = UCS_BIT(13),
+    /* Device supports symmetric key creation */
+    UCT_IB_MLX5_MD_FLAG_MKEY_BY_NAME_RESERVE = UCS_BIT(14),
```

Comment:
maybe just UCT_IB_MLX5_MD_FLAG_MKEY_BY_NAME ?

Reply:
since fw has both features mkey by name, and mkey by name reserve, i'd keep it like that since we use the later feature, let me know otherwise.

Reply:
Is it still the relevant name for the flag?